composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=5.3.0",
```

Comment:
Because there are many people out there still using php 5.3 and by replacing "[]" with "array()"
we get fully 5.3.0 compatible spreadsheet-parser : )